src/decorators/db.ts

```diff
@@ -1,11 +1,15 @@
-import Elysia from "elysia";
-import { DbAction } from "../types/database";
-import Database from "bun:sqlite";
+import Elysia from 'elysia'
+import { DbAction } from 'types/database'
+import Database from 'bun:sqlite'
```

Comment:
I would guessed that this import would be above the types now, with prettier.

Reply:
reordered imports